seqr/models.py

```diff
@@ -1104,6 +1104,35 @@ class Meta:
         json_fields = ['gene_id', 'tpm']
 
 
+class RnaSeqSpliceOutlier(DeletableSampleMetadataModel):
+    STRAND_CHOICES = (
+        ('+', '5′ to 3′ direction'),
+        ('-', '3′ to 5′ direction'),
+        ('*', 'Any direction'),
+    )
+
+    p_value = models.FloatField()
+    z_score = models.FloatField()
+    chrom = models.CharField(max_length=2)
+    start = models.IntegerField()
+    end = models.IntegerField()
+    strand = models.CharField(max_length=1, choices=STRAND_CHOICES)  # "+", "-", or "*"
```

Comment:
are we going to show this in seqr?

Reply:
Yes. Ben mentioned, `hover over the tag should show info about the splice junction outlier (chrom/start/end/strand, z-score, pvalue, maybe tissue and read count also)`.